runtime/criusupport/criusupport.cpp

```diff
@@ -638,17 +636,12 @@ Java_org_eclipse_openj9_criu_CRIUSupport_checkpointJVMImpl(JNIEnv *env,
 	UDATA msgCharLength = 0;
 	IDATA systemReturnCode = 0;
 	const char *dlerrorReturnString = NULL;
-	bool setupCRIU = true;
 	PORT_ACCESS_FROM_VMC(currentThread);
 
-	if (NULL == vm->checkpointState.criuJVMCheckpointExceptionClass) {
```

Comment:
we should assert of throw an exception if this is not true

Reply:
I think we should throw, because someone could call the method via reflection

Reply:
Kept the initial invocation of `setupJNIFieldIDsAndCRIUAPI()` in case `checkpointJVMImpl()` is called via reflection.
`Java_org_eclipse_openj9_criu_CRIUSupport_setupJNIFieldIDsAndCRIUAPI()` is just a wrapper of `setupJNIFieldIDsAndCRIUAPI()`.
@tajila this is ready for another look.